<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compare Images</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="batchLabel">
        <property name="text">
         <string>Batch index (n)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="batchSlider">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="channelLabel">
        <property name="text">
         <string>Channel (c)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="channelSlider">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
      <item>
       <widget class="QLabel" name="label_image1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>17</width>
          <height>28</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Compare</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="batchLabel_2">
        <property name="text">
         <string>Batch index (n)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="batchSlider_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="channelLabel_2">
        <property name="text">
         <string>Channel (c)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="channelSlider_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsView_2"/>
      </item>
      <item>
       <widget class="QLabel" name="label_image2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>253</width>
          <height>28</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1009</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen_Images">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImage_1"/>
   </widget>
   <widget class="QMenu" name="menuCalculation_method">
    <property name="title">
     <string>Calculation method</string>
    </property>
    <addaction name="actionBand"/>
    <addaction name="actionComplete_Image"/>
    <addaction name="actionPixelImage"/>
   </widget>
   <addaction name="menuOpen_Images"/>
   <addaction name="menuCalculation_method"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImage_1">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionImage_2">
   <property name="text">
    <string>Image 2</string>
   </property>
  </action>
  <action name="actionBand">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Band</string>
   </property>
  </action>
  <action name="actionPixel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pixel</string>
   </property>
  </action>
  <action name="actionComplete_Image">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Complete Image</string>
   </property>
  </action>
  <action name="actionPixelImage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pixel</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
